docs/source/templates/class_api.rst

```diff
@@ -145,7 +141,7 @@ Non-Member Functions
 
   :tparam ViewDst: the other
 
-  :return: true if :cpp:type:`View::value_type`, :cpp:type:`View::array_layout`, :cpp:type:`View::memory_space`, :cpp:member:`View::rank`, :cpp:func:`View::data()` and :cpp:expr:`View::extent(r)`, for :cpp:expr:`0<=r<rank`, match.
+  :return: true if :cpp:type:`View::value_type`, :cpp:type:`View::array_layout`, :cpp:type:`View::memory_space`, :cpp:member:`View::func`, :cpp:func:`View::data()` and :cpp:expr:`View::extent(r)`, for :cpp:expr:`0<=r<rank`, match.
```

Comment:
What is `View::func` referring to?

Reply:
That's a typo, meant to be:
```rst
:cpp:func:`View::rank`
```

I'll fix